samples/boards/milkv_duo/mbox/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
```

Comment:
please, no board specific samples for generic APIs, it makes no sense. Also, sample is broken, please, check how the API changed.